OamLog+forwardings.yaml

```diff
@@ -0,0 +1,520 @@
+      operation-client-deletion:
+    initiating-requests:
+    consequent-requests:
+      - client-name: ApplicationLayerTopology://v1/delete-ltp-and-dependents
+        uuid: ol-1-0-0-op-c-bm-alt-1-0-0-002
+
+  - forwarding-name: OamRequestCausesFcUpdateRequest
+    uuid: ol-1-0-0-op-fc-bm-013
+    forwarding-type: InvariantProcessSnippet
+    management-requests:
+      operation-client-update:
+        - server-name: /v1/redirect-topology-change-information
+          uuid: ol-1-0-0-op-s-bm-009
+      fc-port-update:
+        - server-name: /v1/redirect-topology-change-information
+          uuid: ol-1-0-0-op-s-bm-009
+      fc-port-deletion:
+      operation-client-deletion:
+    initiating-requests:
+    consequent-requests:
+      - client-name: ApplicationLayerTopology://v1/update-fc
+        uuid: ol-1-0-0-op-c-bm-alt-1-0-0-003
+
+  - forwarding-name: OamRequestCausesFcPortUpdateRequest
+    uuid: ol-1-0-0-op-fc-bm-014
+    forwarding-type: InvariantProcessSnippet
+    management-requests:
+      operation-client-update:
+        - server-name: /v1/redirect-topology-change-information
+          uuid: ol-1-0-0-op-s-bm-009
+      fc-port-update:
+        - server-name: /v1/redirect-topology-change-information
+          uuid: ol-1-0-0-op-s-bm-009
+      fc-port-deletion:
+      operation-client-deletion:
+    initiating-requests:
+    consequent-requests:
+      - client-name: ApplicationLayerTopology://v1/update-fc-port
+        uuid: ol-1-0-0-op-c-bm-alt-1-0-0-004
+
+  - forwarding-name: OamRequestCausesFcPortDeletionRequest
+    uuid: ol-1-0-0-op-fc-bm-015
+    forwarding-type: InvariantProcessSnippet
+    management-requests:
+      operation-client-update:
+        - server-name: /v1/redirect-topology-change-information
+          uuid: ol-1-0-0-op-s-bm-009
+      fc-port-update:
+        - server-name: /v1/redirect-topology-change-information
+          uuid: ol-1-0-0-op-s-bm-009
+      fc-port-deletion:
+      operation-client-deletion:
+    initiating-requests:
+    consequent-requests:
+      - client-name: ApplicationLayerTopology://v1/delete-fc-port
+        uuid: ol-1-0-0-op-c-bm-alt-1-0-0-005
+
+  - forwarding-name: NewApplicationCausesRequestForOamRequestInformation
+    uuid: ol-1-0-0-op-fc-is-000
+    forwarding-type: ProcessSnippet
+    management-requests:
+      operation-client-update:
+        - server-name: /v1/regard-application
+          uuid: ol-1-0-0-op-s-is-001
+      fc-port-update:
+        - server-name: /v1/regard-application
+          uuid: ol-1-0-0-op-s-is-001
+      fc-port-deletion:
+        - server-name: /v1/disregard-application
+          uuid: ol-1-0-0-op-s-is-002
+      operation-client-deletion:
+        - server-name: /v1/disregard-application
+          uuid: ol-1-0-0-op-s-is-002
+    initiating-requests:
+        - server-name: /v1/regard-application
+          uuid: ol-1-0-0-op-s-is-001
+    consequent-requests:
+      - client-name: RegistryOffice:://v1/redirect-oam-request-information
```

Comment:
uuid of RegistryOffice:://v1/redirect-oam-request-information to be updated to ol-1-0-0-op-c-im-ro-1-0-0-004

Reply:
Changed as proposed. Many thanks for indicating the mistake.

Reply:
Somehow the update is not reflecting for RegistryOffice:://v1/redirect-oam-request-information